api/README.md

```diff
@@ -32,6 +46,16 @@ You will also need to create a SQL API database named **2wr** and the following
 | hazardhunts | /id |
 | familyplan | /userId |
 
+Alternately, you can run the tool in:
+
+```
+> cd ..\tools\CosmosEmulator
+
+> dotnet run /setup
+```
+
+This will create the database and collections for you.
+
 ### Create a local.settings.json
 
 Create a `local.settings.json` file in the TwoWeeksReady project folder. You will need to set the CosmosDbConnection setting to your own CosmosDB instance.
```

Comment:
Will folks also need to add a storage connection string to local.settings.json?

Reply:
Yes, let me add that.

Reply:
Added...